src/environment_setup/tool_providers.ml

```diff
@@ -213,6 +213,18 @@ let freebayes =
         KEDSL.Program.(shf "export PATH=%s/bin/:$PATH" path))
     ~witness:(witness_list ["bin/freebayes"; "bin/bamleftalign"])
 
+let delly2 =
+  Installable_tool.make
+    Machine.Tool.Default.delly2
+    ~url:"https://github.com/dellytools/delly/releases/download/v0.7.7/delly_v0.7.7_parallel_linux_x86_64bit"
```

Comment:
"Delly2 supports parallel computing using the OpenMP API", so, since it works, might as well build this one depending on the backend one might use?